RecoHGCal/TICL/test/ticl_iterations.py

```diff
@@ -0,0 +1,115 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoLocalCalo.HGCalRecProducers.HGCalUncalibRecHit_cfi import *
+from RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi import *
+
+# patch particle flow clusters for HGC into local reco sequence
+# (for now until global reco is going with some sort of clustering)
+from RecoParticleFlow.PFClusterProducer.particleFlowRecHitHGC_cff import *
+from RecoParticleFlow.PFClusterProducer.particleFlowClusterHGC_cfi import *
+from RecoLocalCalo.HGCalRecProducers.hgcalLayerClusters_cff import hgcalLayerClusters
+from RecoLocalCalo.HGCalRecProducers.hgcalMultiClusters_cfi import hgcalMultiClusters
+
+
+from RecoHGCal.TICL.Tracksters_cfi import Tracksters
+from RecoHGCal.TICL.FilteredLayerClusters_cfi import FilteredLayerClusters
+from RecoHGCal.TICL.TrackstersToMultiCluster_cfi import TrackstersToMultiCluster
+
+
+def TICL_iterations_withReco(process):
+  process.FEVTDEBUGHLTEventContent.outputCommands.extend(['keep *_TrackstersToMultiCluster*_*_*'])
+
+  process.FilteredLayerClustersMIP = FilteredLayerClusters.clone()
+  process.FilteredLayerClustersMIP.ClusterFilter = "ClusterFilterBySize"
+  process.FilteredLayerClustersMIP.algo_number = 8
+  process.FilteredLayerClustersMIP.max_cluster_size = 2 # inclusive
+  process.FilteredLayerClustersMIP.iteration_label = "MIP"
+  process.TrackstersMIP = Tracksters.clone()
+  process.TrackstersMIP.original_layerclusters_mask = cms.InputTag("hgcalLayerClusters", "InitialLayerClustersMask")
```

Comment:
it's a quite a bit less verbose to specify all the module parameters that differ from the origin in the parentheses of the ```clone``` method call.